testcases/OpTestEEH.py

```diff
@@ -196,6 +229,7 @@ def check_eeh_slot_resets(self):
         cmd = "cat /proc/powerpc/eeh | tail -n 1"
         count = self.cv_SYSTEM.sys_get_ipmi_console().run_command(cmd)
         output = (count[-1].split("="))[1]
+        print output
```

Comment:
Why do we need to print output? Is what we throw in the exception / fail the test with incomplete?

It would be better if we fail the test with *all* the applicable error messages, as that way they'll show up properly in the  test runner logs